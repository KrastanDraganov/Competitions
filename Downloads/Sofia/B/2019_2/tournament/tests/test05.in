50
15 6 7 9 11 13 14 18 22 26 34 36 39 40 43 48
24 0 2 4 6 7 9 11 14 16 18 22 23 27 31 32 34 36 39 40 41 43 45 47 48
23 0 4 6 7 8 9 11 14 16 18 22 27 31 32 34 36 39 40 41 43 44 47 48
37 0 1 2 6 7 8 9 10 11 14 15 16 17 18 19 22 23 24 26 27 28 29 30 31 32 34 36 37 38 39 41 43 44 46 47 48 49
18 0 3 6 7 9 11 13 14 18 22 32 34 36 39 40 43 47 48
49 0 1 2 3 4 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49
5 10 14 25 35 43
14 6 9 11 13 14 18 22 34 36 37 39 40 43 48
24 0 1 4 6 7 9 11 13 14 18 20 22 27 32 34 35 36 38 39 40 41 43 47 48
9 6 14 18 22 32 34 36 39 43
39 0 1 2 4 7 8 9 11 13 14 15 17 18 19 21 22 23 24 25 26 27 28 29 30 31 32 34 36 37 38 39 40 41 43 44 46 47 48 49
10 6 9 14 18 22 34 36 39 41 43
42 0 1 2 3 4 6 7 8 9 10 11 13 14 15 16 17 18 19 21 23 24 25 26 27 28 29 30 31 32 34 36 37 38 39 40 41 42 44 45 46 47 48
15 1 2 3 6 9 11 14 18 20 22 34 36 39 40 43
2 43 44
27 0 1 2 4 6 7 8 9 11 13 14 16 18 21 22 23 27 32 34 36 38 39 40 41 43 47 48
23 0 4 6 7 8 9 10 11 13 14 18 22 27 31 32 34 36 39 40 41 43 47 48
32 0 1 2 4 6 7 8 9 11 13 14 15 16 18 21 22 23 27 28 31 32 34 36 38 39 40 41 43 46 47 48 49
4 6 14 43 45
37 0 1 2 4 6 7 8 9 11 13 14 15 16 17 18 21 22 23 26 27 28 29 30 31 32 34 36 38 39 40 41 43 44 46 47 48 49
40 0 1 2 3 4 6 7 9 10 11 12 14 15 16 17 18 19 21 22 23 24 25 26 27 28 30 31 32 34 36 38 39 40 41 43 44 46 47 48 49
27 0 1 2 3 4 6 7 8 9 11 13 14 16 18 22 23 27 31 32 34 36 38 39 40 41 43 47
10 6 12 14 18 26 34 36 39 43 45
25 0 2 4 6 7 8 9 11 13 14 16 18 22 27 31 32 34 36 37 39 40 41 43 47 48
38 0 1 2 4 6 7 8 9 11 13 14 15 16 17 18 19 21 22 23 25 26 27 28 29 30 31 32 36 37 38 39 40 43 44 46 47 48 49
39 0 1 2 3 4 7 8 9 11 13 14 15 16 17 18 19 21 22 23 26 27 28 29 30 31 32 34 36 37 38 39 40 41 43 44 46 47 48 49
32 1 2 4 6 7 8 9 11 13 14 15 16 17 18 21 23 27 28 29 30 31 32 34 36 39 40 41 43 46 47 48 49
19 0 4 6 7 9 11 13 14 18 22 32 34 36 38 39 40 43 47 48
30 0 1 2 4 6 7 8 9 11 13 14 15 16 18 21 22 23 27 31 32 34 36 38 39 40 41 43 47 48 49
33 0 1 2 4 6 7 8 9 11 13 14 15 16 17 18 20 21 22 23 27 28 31 32 34 36 38 39 40 41 46 47 48 49
33 0 1 2 4 6 7 8 9 11 13 14 15 16 17 18 21 22 23 27 28 29 31 32 34 36 38 39 40 43 46 47 48 49
20 0 4 6 7 8 9 11 13 14 15 18 22 27 32 34 36 39 41 43 48
15 0 6 7 11 13 14 18 22 34 36 39 40 43 47 48
47 0 1 2 3 4 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 34 35 36 37 38 39 40 41 43 44 45 46 47 48 49
6 6 14 18 24 39 43
45 0 1 2 3 4 7 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 34 36 37 38 39 40 41 42 43 44 45 46 47 48 49
6 6 14 18 34 39 43
37 0 1 2 4 6 8 9 11 13 14 15 16 17 18 19 20 21 22 26 27 28 29 30 31 32 34 36 38 39 40 41 43 44 46 47 48 49
24 0 1 2 4 6 7 9 11 13 14 16 18 22 23 26 31 32 34 36 39 40 41 43 47
5 6 14 18 43 44
12 3 6 9 11 14 18 22 31 34 36 39 44
20 0 4 6 7 9 13 14 18 22 24 27 30 32 34 36 39 40 43 47 48
44 0 1 2 3 4 6 7 8 9 10 11 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 36 37 38 39 40 41 43 46 47 48 49
3 12 29 40
33 0 1 4 6 7 8 9 11 13 15 16 17 18 21 22 23 26 27 28 29 30 31 32 34 36 38 41 42 43 46 47 48 49
42 0 2 3 4 6 7 8 9 10 11 13 14 15 16 17 19 20 21 23 24 25 26 27 28 29 30 31 32 34 36 37 38 39 40 41 42 43 44 46 47 48 49
31 0 1 2 4 6 7 8 9 11 13 14 15 16 18 21 22 23 27 28 31 32 34 36 38 39 40 41 43 47 48 49
16 0 6 7 9 11 13 14 18 22 31 34 36 39 40 43 48
14 6 9 11 13 14 18 21 22 34 36 38 39 40 43
30 0 1 2 4 6 7 8 9 11 12 13 14 15 16 18 21 22 23 27 31 32 34 36 38 39 40 41 43 47 48
