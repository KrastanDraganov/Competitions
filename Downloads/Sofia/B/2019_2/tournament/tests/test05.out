5 45 44 42 33 35 37 46 49 41 30 29 20 48 38 47 40 36 34 24 25 26 28 23 31 27 32 22 12 19 17 15 21 16 8 4 13 3 1 2 0 7 9 39 18 6 10 11 14 43 
